scripts/generate_can_enums.py

```diff
@@ -55,7 +55,10 @@ def canid_filters_formatted(canid_filters):
 
 
 def roboteq_enums_formatted(roboteq_enums):
```

Comment:
nit: these types of functions should be named as `format_roboteq_enums` or similar imperative phrasing. the way it is now should be reserved for properties